tests/foreman/api/test_subnet.py

```diff
@@ -32,119 +40,170 @@
-
         :id: b3de6f96-7c39-4c44-b91c-a6d141f5dd6a
-
         :steps:
-
             1. Create Subnet with all the details
             2. Create subnet parameter having single key and multiple values
                 separated with comma
-
         :expectedresults: The parameter with multiple values should be saved
             in subnet
         """
+        subnet = entities.Subnet().create()
+        name = gen_string('alpha')
+        values = ', '.join(generate_strings_list(exclude_types=['html']))
+        with self.subTest(name):
+            subnet_param = entities.Parameter(
+                name=name,
+                subnet=subnet.id,
+                value=values
+            ).create()
+            self.assertEqual(subnet_param.name, name)
+            self.assertEqual(subnet_param.value, values)
 
-    @stubbed()
     @tier1
     def test_positive_create_with_parameter_and_valid_separator(self):
         """Subnet parameters can be created with multiple names with valid
         separators
-
         :id: d1e2d75a-a1e8-4767-93f1-0bb1b75e10a0
-
         :steps:
-
             1. Create Subnet with all the details
             2. Create subnet parameter having key with multiple names separated
                 by valid separators(e.g fwd slash) and value
-
         :expectedresults: The parameter with multiple names separated by valid
             separators should be saved in subnet
         """
+        subnet = entities.Subnet().create()
+        valid_separators = [',', '/', '-', '|']
+        valid_names = []
+        for separator in valid_separators:
+            valid_names.append(
+                '{}'.format(separator).join(generate_strings_list(
+                    exclude_types=['html']))
+            )
+        value = gen_string('utf8')
+        for name in valid_names:
+            with self.subTest(name):
+                subnet_param = entities.Parameter(
+                    name=name,
+                    subnet=subnet.id,
+                    value=value
+                ).create()
+                self.assertEqual(subnet_param.name, name)
+                self.assertEqual(subnet_param.value, value)
 
-    @stubbed()
     @tier1
     def test_negative_create_with_parameter_and_invalid_separator(self):
         """Subnet parameters can not be created with multiple names with
         invalid separators
-
         :id: 08d10b75-a0db-4a11-a915-965a2a207d16
-
         :steps:
-
             1. Create Subnet with all the details
             2. Create subnet parameter having key with multiple names separated
                 by invalid separators(e.g spaces) and value
-
         :expectedresults:
-
             1. The parameter with multiple names separated by invalid
                 separators should not be saved in subnet
             2. An error for invalid name should be thrown.
         """
+        subnet = entities.Subnet().create()
+        invalid_values = invalid_values_list() + ['name with space']
```

Comment:
same here
It is valid to say that parameter cannot have value with invalid separator between words or just cannot have value that contains specific characters. But still not sure what word 'multiple' is doing here

Reply:
Removed multiple word.